### PR TITLE
doc: reword mds deactivate docs; add optional fs_name argument

### PR DESCRIPTION
**Pull request by jan--f**
Fixes: http://tracker.ceph.com/issues/20607

Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins retest this please

File: doc/cephfs/administration.rst [hunk 1]
```diff
@@ -94,12 +94,13 @@ available, then the "failed" daemon will return as a standby.
 
 ::
 
-    mds deactivate <role>
+    mds deactivate [<fs_name>:]<rank>
```
Comment (batrick) [inline doc/cephfs/administration.rst]: The `role` argument can actually be other forms too;

https://github.com/jan--f/ceph/blob/2155f7fef6415ac0cca88e1eb5a7b12c995f3433/src/mds/FSMap.cc#L970-L982

I'd prefer leaving it as role and link to a doc explaining what a role's form can be.

File: doc/cephfs/administration.rst [hunk 2]
```diff
@@ -94,12 +94,13 @@ available, then the "failed" daemon will return as a standby.
...
 
-Deactivate an MDS, causing it to flush its entire journal to
-backing RADOS objects and close all open client sessions. Deactivating an MDS
-is primarily intended for bringing down a rank after reducing the number of
-active MDS (max_mds).
+Deactivating a rank causes the MDS daemon servicing this rank to flush its
+entire journal to the backing RADOS objects and close all open client
+sessions. Deactivating a rank is primarily intended to have an MDS daemon
+rejoin the cluster as a standby after reducing the number of active MDS
```
Comment (batrick) [inline doc/cephfs/administration.rst]: The previous wording is correct. The actual intent is to shrink the number of active ranks. You can add that the MDS will rejoin as a standby after deactivating the rank.

File: doc/cephfs/multimds.rst [hunk 3]
```diff
@@ -81,7 +81,7 @@ Note that we still have two active MDSs: the ranks still exist even though
 we have decreased max_mds, because max_mds only restricts creation
 of new ranks.
 
-Next, use the ``ceph mds deactivate <rank>`` command to remove the
+Next, use the ``ceph mds deactivate <fs_name>:<rank>`` command to remove the
```
Comment (batrick) [inline doc/cephfs/multimds.rst]: Use `<role>`.

**Comment by jan--f** [comment]
@batrick Thank you for clarifying!

**Pull request merged**